tests/api/test_api.py

```diff
@@ -24,6 +25,20 @@ def test_get_zones(self):
 
         assert response[0]["id"] == 1
 
+    def test_get_boiler_state(self):
```

Comment:
Add a condition to allow running the unit test without the `TADO_BRIDGE_AUTHKEY` envvar, which will be 99% of the times in the CI/